regolith/helpers/u_milestonehelper.py

```diff
@@ -65,13 +66,10 @@ def construct_global_ctx(self):
     def db_updater(self):
         rc = self.rc
         key = rc.projectum_id
-        coll = self.gtx[rc.coll]
-        pdocl = list(filter(lambda doc: doc["_id"] == key, coll))
-        if len(pdocl) == 0:
-            raise RuntimeError(
-                "This entry appears to not exist in the collection")
         filterid = {'_id': key}
         current = rc.client.find_one(rc.database, rc.coll, filterid)
```

Comment:
can we change this variable name.  A better name describes what it is.  Like "target_prum" or sthg.

Reply:
done!